src/block_service/fds/fds_service.h

```diff
@@ -110,6 +110,7 @@ class fds_file_object : public block_file
                                    dsn::task_code code,
                                    const download_callback &cb,
                                    dsn::task_tracker *tracker) override;
+    error_code get_file_meta();
```

Comment:
I suggest adding comments to show the possible return error code.

Reply:
```suggestion
    // Possible errors:
    // - ERR_OBJECT_NOT_FOUND
    // - ERR_FS_INTERNAL
    error_code get_file_meta();
```